### ISSUE TITLE
Frequently used components should be well managed for reusability

### ISSUE THREAD

**Issue by sleepylee** [longngo1313/Nimbl3_VuLong#3]
It's common that a component style could appear several times in an application, for example, this button style:

<img src="https://user-images.githubusercontent.com/13435717/29549042-d8ea3f08-872f-11e7-8df3-a83d30d23f18.png" width=200/>

 it should be better initiated for reusability later without replicating the styling properties.

**Comment by longngo1313** [comment]
Yes, i used initiated style and theme as an collection of attributes in my projects. However, this application is very simple with only one button. It make me a little confuse to pick what attributes should implement in the style. So, i just do not implement it on my code.
